packages/relic/lib/src/headers/typed/headers/content_encoding_header.dart

```diff
@@ -93,7 +93,7 @@ class ContentEncoding {
       case _zstd:
         return zstd;
       default:
-        return ContentEncoding(trimmed);
+        throw FormatException('Invalid value');
```

Comment:
Question: Was the default case changed to throw intentional?

Reply:
We want only to accept headers and values that are known, not custom values.